Xamarin.Forms.Material.Android/MaterialButtonRenderer.cs

```diff
@@ -284,8 +284,8 @@ void UpdatePrimaryColors()
 			else
 				text = textColor.ToAndroid();
 
-			// apply
-			SetTextColor(MaterialColors.CreateButtonTextColors(background, text));
+			// apply			
+			SetTextColor(MaterialColors.CreateButtonTextColors(background, text, Element.DisabledTextColor.ToAndroid()));
```

Comment:
Should checked if DisabledTextColor is the default color, and in that case use a disable text color from MaterialColors.